js/index.js

```diff
@@ -36,3 +36,77 @@ for (let i = 0; i < skills.length; i++) {
   // Append the list item element to the unordered list
   skillsList.appendChild(skill);
 }
+
+// Select the message form by name
+const messageForm = document.forms.leave_message;
+
+// Add event listener for form submit
+messageForm.addEventListener('submit', function(event) {
+  // Prevent the form from submitting normally
+  event.preventDefault();
+
+  // Retrieve the form field values
+  const name = event.target.usersName.value;
+  const email = event.target.usersEmail.value;
```

Comment:
Should use let for these.  They are set each time the callback function is called.  const variables are intended to be set once.

Reply:
fixed